axon.go

```diff
@@ -143,19 +139,15 @@ func (a *Axon) Run() {
 			wp.Close()
 			sourceDBConn.Close()
 			targetDBConn.Close()
-			return
+			return nil
 		case err := <-errs:
-			a.Logger.WithError(err).
-				WithField("component", "warp_pipe").
-				Error("received an error")
+			return fmt.Errorf("listener received an error: %w", err)
 		case change := <-changes:
 			a.processChange(sourceDBConn, targetDBConn, a.Config.TargetDBSchema, change)
```

Comment:
To be honest, I am not sure why it was like this either. @perangel was this intentional?

Reply:
Agreed stop on error

Reply:
Yes, not stopping on error was intentional, but it may not suit the broader use case, or just require refinement so that you could control what should/n't be fatal.